main.go

```diff
@@ -307,7 +307,7 @@ func validateParams() providerConfig {
 	var ecrEnabled bool
 
 	awsAccountID = os.Getenv("awsaccount")
-	if len(awsAccountID) == 0 {
+	if len(awsAccountID) == 0 || awsAccountID == "changeme" {
```

Comment:
Ok I can work on moving that into a configmap, and cleaning up the code, unless you've got something started or want to tackle. 

I want to get this working @tsloughter first in this repo, then will look to incorporate into minikube which is what we had originally discussed.